rest-api-spec/src/main/resources/rest-api-spec/api/esql.query.json

```diff
@@ -29,6 +29,11 @@
         "type":"string",
         "description":"The character to use between values within a CSV row. Only valid for the csv format.",
         "default":false
+      },
+      "drop_null_columns": {
+        "type": "boolean",
+        "description": "Should null columns be removed from the main output and reported in a special null_columns section?",
```

Comment:
"... removed from the columns section. The full query output is reported through the dedicated 'all_columns' section"

Or something like that.